### DISCUSSION TITLE
Regenerate Onboarding PR

### DISCUSSION THREAD

**Discussion by StuStirling** [renovatebot/renovate#19053] [Request Help]
I installed the app prematurely, before migrating my dependencies to gradle version catalogs. I tried to fix it by rebasing the rennovate branch but i don't think it's worked. I've since renamed the rennovate branch by appending `- old`. 

Is there any chance you could trigger a new onboarding PR?

**Comment by viceice** [comment]
restore the branch, open the pr and tick the rebase checkbox

**Reply by StuStirling** [reply]
Thanks I've been through the onboarding and merged it into my default branch. Problem is, I then deleted the onboarding pull request and its branch and now I'm told it's disabled the renovate app. How do i get this enabled again?

**Reply by rarkins** [reply]
We may need to help you manually. If you know a past "job ID" you can tell that here, or the repo name. Feel free to email it to me if you don't want to say it publicly

**Reply by StuStirling** [reply]
Here's the latest job id that disabled renovate - `901506395`

**Reply by StuStirling** [reply]
Looking at the logs it can't find my `renovate.json` file but its in `.github/renovate.json` in the base branch `develop` which is defined in the config `"baseBranches": ["develop"]`

**Reply by rarkins** [reply]
You must have a Renovate config file in the default branch. If you don't have that then it's a config problem. Check out `useBaseBranchConfig` if you wish to have the config in your base branch override the config in your default

**Reply by rarkins** [reply]
Let me know once you fix that and I can then trigger a new job if one didn't run automatically

**Reply by StuStirling** [reply]
Ok, I've just merged my `develop` branch to `main` so it now has the `renovate.json` file in there

**Reply by StuStirling** [reply]
And it looks like its created theWas triggered automatically 👍
